pkg/github/github.go

```diff
@@ -5,10 +5,34 @@ import (
 	"fmt"
 	"log"
 	"strings"
+	"time"
 
+	"github.com/cyberark/conjur-oss-suite-release/pkg/changelog"
 	"github.com/cyberark/conjur-oss-suite-release/pkg/http"
+	"github.com/cyberark/conjur-oss-suite-release/pkg/repositories"
+	"github.com/cyberark/conjur-oss-suite-release/pkg/version"
 )
 
+// SuiteComponent represents a suite component with all of its changelogs and
+// relevant pin data
+type SuiteComponent struct {
+	CertificationLevel string
+	Changelogs         []*changelog.VersionChangelog
+	ReleaseName        string
+	ReleaseDate        string
+	Repo               string
+	UpgradeURL         string
+	URL                string
+}
```

Comment:
This doesn't seem to fit in this pkg cleanly but I don't have a better solution other than having it in template too. I guess it can stay as-is but we should look into the right home for this structure later.

Reply:
Agreed. I have to wonder if there's value in having another package that handles the 'in-between' work of communicating github and parsing a template, where things like SuiteComponent and its relevant methods that modify the data we already have make more sense.